service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceOperationsTest.java

```diff
@@ -651,7 +619,7 @@ public void invokeSucceed(
                 httpRequest.setReadTimeoutMillis(DEFAULT_HTTP_TIMEOUT_MS);
                 times = 1;
                 httpRequest.setHeaderField(anyString, anyString);
-                times = 5;
+                times = 4;
```

Comment:
Why this change?

Reply:
Some of the mocking in the unit test broke when I switched to validating the requestId parameter by calling Tools.isNullOrEmpty, so the header case for adding a request Id header isn't called from this unit test anymore